src/components/interfaces/HMI_API.xml

```diff
@@ -2638,7 +2638,7 @@
     <param name="vrSynonyms" type="String" maxlength="40" minsize="1" maxsize="100" array="true" mandatory="false">
       <description>
         Defines an additional voice recognition command.
-        Must not interfere with any name of previously registered applications(SDL makes check).
+        Must not interfere with any name of previously registered applications from the same device.
```

Comment:
Updated `vrSynonyms` description should also apply to `VR.ChangeRegistration` and `BasicCommunication.OnAppRegistered`

Reply:
@ShobhitAd  fixed 862e9185dfc3449002c037cffaeb500d58642b18